lib/internal/test_runner/utils.js

```diff
@@ -168,7 +169,10 @@ async function setupTestReporters(rootTest) {
   const reportersMap = await getReportersMap(reporters, destinations, rootTest);
   for (let i = 0; i < reportersMap.length; i++) {
     const { reporter, destination } = reportersMap[i];
-    compose(rootTest.reporter, reporter).pipe(destination);
+    compose(rootTest.reporter, reporter).on('error', (err) => {
+      process.exitCode ||= 1;
+      console.error(err);
```

Comment:
nit, this should _probably_ be wrapped in a try/catch for robustness to guard against weirdness like:

```
new class extends Error { get message() { throw 'up'; }}
```